src/app/components/RelatedProducts/RelatedProducts.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Carousel from '../../shared/Carousel/index';
+import ProductItemContainer from '../../shared/ProductItem/ProductItemContainer';
+
+import './RelatedProducts.scss';
+
+const title = 'Related Products';
+
+const RelatedProducts = props => {
+  const titleArr = title.split(' ');
+  const { products, wishlist } = props;
+  const extended = true;
+  const isAddedtoWishList = id => wishlist.includes(id);
+
+  const list =
+    products &&
+    products.map(el => (
+      <ProductItemContainer
+        isAddedtoWishList={isAddedtoWishList(el._id)}
+        extended={extended}
+        key={el._id}
+        data={el}
+      />
+    ));
+
+  return (
+    <section className="container related-products">
+      <div className="section_heading">
+        <h2>
+          <span className="highlighted">{titleArr[0]}</span>
+          &nbsp;
+          {titleArr[1]}
+        </h2>
+        <p>
+          Lorem Ipsum is simply dummy text of the printing and typesetting
+          industry
+        </p>
+      </div>
+      <Carousel data={products} extended>
+        {list}
+      </Carousel>
+    </section>
+  );
+};
+
+RelatedProducts.propTypes = {
+  products: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Not the best way to describe products.
I have created actual file which describes single product shape in _types_ folder

Reply:
Also I would use isRequired instead of empty default arrays

Reply:
@moskdev https://medium.freecodecamp.org/react-pattern-centralized-proptypes-f981ff672f3b consider this manual